### PR TITLE
Feature/output format

### PR DESCRIPTION
**Pull request by ChakavehSaedi**

### CODE REVIEW

File: relevanceai/http_client.py [hunk 1]
```diff
@@ -52,8 +52,15 @@ def __init__(
             self.projector = Projector(project, api_key)
 
         self.vector_tools = VectorTools(project, api_key)
+        self.output_format = CONFIG.get_field("api.output_format", CONFIG.config)
```
Comment (boba-and-beer) [inline relevanceai/http_client.py:55]: Remove this

File: tests/unit/test_config.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import pytest
+
+def test_output_format(test_client, test_sample_dataset):
+    import random
```
Comment (boba-and-beer) [inline tests/unit/test_config.py:4]: move to the top

File: tests/unit/test_config.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            {
+                "vector": [random.randint(0, 1000) for _ in range(100)],
+                "fields": ["sample_1_vector_"]
+            }
+        ]
+    )
+    assert isinstance(results, dict)
+
+    # Content output
+    test_client.output_format = "content"
```
Comment (boba-and-beer) [inline tests/unit/test_config.py:20]: Separate this into a separate test. Try to only have 1 assert each test.

Comment (boba-and-beer) [reply tests/unit/test_config.py:20]: Look into @pytest.mark.parametrize if you want to make it easier for yourself :)

**Review by boba-and-beer — CHANGES_REQUESTED** [review]

File: relevanceai/http_client.py (hunk 1 above)
Comment (ChakavehSaedi) [reply relevanceai/http_client.py]: Done!

File: tests/unit/test_config.py (hunk 2 above)
Comment (ChakavehSaedi) [reply tests/unit/test_config.py]: Done!

File: tests/unit/test_config.py (hunk 3 above)
Comment (ChakavehSaedi) [reply tests/unit/test_config.py]: Done!
Good learn - Thanks

**Pull request merged**